website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -100,10 +100,28 @@ spec:
       serviceAccountName: static-server
 ```
 
-## Known Beta Limitations
+### Kubernetes HTTP Health Probes Configuration
+Traffic redirection interferes with [Kubernetes HTTP health
+probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/) since the
+probes expect that kubelet can directly reach the application container on the probe's endpoint, but that traffic will
+be redirected through the sidecar proxy, causing errors because kubelet itself is not encrypting that traffic using a
+mesh proxy. For this reason, Consul allows you to overwrite Kubernetes HTTP health probes to point to the proxy instead,
+to pass that traffic through via the proxy using the Helm value `connectInject.transparentProxy.defaultOverwriteProbes`
+or the Pod annotation `consul.hashicorp.com/transparent-proxy-overwrite-probes`.
 
-* There is no first class support for transparent proxying on VMs. 
-* Traffic can only be transparently proxied when the address dialed corresponds to the address of a service in the 
+### Traffic Redirection Configuration
+Pods with transparent proxy enabled will have an init container injected that sets up traffic redirection for all
+inbound and outbound traffic through the sidecar proxies. This will include all traffic by default, with the ability to
+configure exceptions on a per-Pod basis. The following Pod annotations allow you to exclude certain traffic from redirection to the sidecar proxies:
+- [`consul.hashicorp.com/transparent-proxy-exclude-inbound-ports`](/docs/k8s/connect#consul-hashicorp-com-transparent-proxy-exclude-inbound-ports)
+- [`consul.hashicorp.com/transparent-proxy-exclude-outbound-ports`](/docs/k8s/connect#consul-hashicorp-com-transparent-proxy-exclude-outbound-ports)
+- [`consul.hashicorp.com/transparent-proxy-exclude-outbound-cidrs`](/docs/k8s/connect#consul-hashicorp-com-transparent-proxy-exclude-outbound-cidrs)
+- [`consul.hashicorp.com/transparent-proxy-exclude-uids`](/docs/k8s/connect#consul-hashicorp-com-transparent-proxy-exclude-uids)
```

Comment:
It would nice if we had one lines here that describe what the annotation does with an example value or two.

Reply:
Yes please! cc @ishustava

Reply:
great idea @thisisnotashwin! let me update my PR to include those since Nitya is just linking to those annotations from here.